src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -101,17 +92,19 @@
 
     /**
      * This is our {@link Thread} pool.
-     * Using an {@link ExecutorService} here instead of continously spawning a new {@link Thread}
+     * Using an {@link ExecutorService} here instead of continuously spawning a new {@link Thread}
      * will help performance.
      */
     private final ExecutorService asyncExecutor = Executors.newSingleThreadExecutor(threadFactory);
 
+
     /**
-     * Making collections concurrent doesn't make their iterators thread safe.
+     *  These are "Queues" of blocks that need to be removed or moved.
+     *  <p>
+     *  Whilst the {@link ConcurrentHashMap} implementation is thread-safe,
+     *  the {@link Location Locations} are not. Therefore, the internal copy of the locations
+     *  should be treated as <strong>read only</strong> and never exposed externally.
```

Comment:
2 spaces at the start here
```suggestion
     * These are "Queues" of blocks that need to be removed or moved.
     * <p>
     * Whilst the {@link ConcurrentHashMap} implementation is thread-safe,
     * the {@link Location Locations} are not. Therefore, the internal copy of the locations
     * should be treated as <strong>read only</strong> and never exposed externally.
```